clash-protocols/src/Protocols/Df.hs

```diff
@@ -898,6 +898,26 @@ registerBwd =
     iDatX1 = C.regEn (C.errorX "registerBwd") oAck iDatX0
     oDat = toData <$> valid <*> (C.mux oAck iDatX0 iDatX1)
 
+blockRamUNoClear ::
```

Comment:
Perhaps copy+paste the documentation of `blockRamU` and explain that it removes the clear logic.

Reply:
Luckily I don't actually _export_ this identifier, I wouldn't want to do that. The only reason I don't use a local definition is that Fourmolu would get mad at me. The only way to `#if` an expression or definition with Fourmolu is to create a top-level binder for the expression like I do here. You can't do `#if` in the middle of an expression and you can't do `#if` with a local definition (`let` or `where` block, although frankly I've never tried a `let`, it seems unlikely it would work).

So I can add some explanation, but it would not affect the generated Haddock or the export list of this module.

Reply:
Ahh, ok. Yeah a quick note saying that this is a Fourmolu workaround is fine. (Or disable Fourmolu locally if it's too much of a hassle?)